### ISSUE TITLE
Using one water bucket per flour sack is repetative

### ISSUE THREAD

**Issue by Heptagon-ru** [anegostudios/VintageStory-Issues#156]
1.6 pre2, pre3, public test server.

- [x] 1. When placing some grain into quern by clicking on it first in inventory and second in quern GUI, it actually doesnt go there, only is shown as if it is there. Aka ghosting. As soon as you click on the inventory slot the grain was in, or do some other action in inventory, it appears again in inventory.
The only way I was able to place grain into quern properly - was to shift-click it.

- [x] 2. When some flour has been produced, it is impossible to remove it from quern into iventory by neither carrying with mouse or shift-clicking. This only results in appearing a "ghost" flour in inventory, disappearing when you try to use it later, and "returning" to quern".
The only way I was able to remove the flour from the quern was to destroy quern.

- [ ] 3. A minor suggestion. I feel using one water bucket per one flour sack is too much. Forces me to do 64 repetative boring action sequences to get 64 dough. I propose 8 or 16 flour sacks per water bucket.
Also when making bread I already have payed with flour making time, maybe no need to charge more price for bread.
Grain grinding is also boring, but it has a very nice animation and sound, so worth it :D

**Comment by tyronx** [comment]
The inventory issues are fixed in pre4. The balance of bread production of it is still an issue

**Comment by Heptagon-ru** [comment]
Wow, a list with checkboxes, awesome! :D

**Comment by copygirl** [comment]
For the record, the suggestion bit could've gone in its own issue, so it could be independently fixed and discussed. As you can see the title and tagging is getting a bit confusing as a result as well. I don't think anyone is bothered by this though so it's not a huge problem.

**Comment by Heptagon-ru** [comment]
I agree 100%, sorry for that.
Kinda thought this was intended, as with cutting cattails, so did not expect much reaction :-P (the tongue is for Tyron).
Yeah, me Englese isant tu well.

**Comment by tatsu011** [comment]
There is now a small patch coming to address the third check-mark.  Will check (or someone else will) once the Pull request is merged.

**Comment by tyronx** [comment]
Hm, I would prefer to add fill levels to water buckets instead of a 3x3 recipe @tatsu011

**Comment by tyronx** [comment]
Ah well, i merge your changes until we got fill levels!

**Comment by copygirl** [comment]
Okay, then it makes sense to keep this issue open as a tracking issue for when the intended crafting recipe is implemented?

**Comment by tatsu011** [comment]
Yeah, I meant my patch to hold over as a temporary fix.

**Comment by tyronx** [comment]
Closing this for now, as we got a temporary fix.

**Comment by tyronx** [comment]
"Aww why did you close the water bucket one? Did you read my last comment?" - copygirl :P

**Comment by tyronx** [comment]
Fill levels are now implemented. You can now use any combinations of flour items + 1 water bucket.
